src/stake.rs

```diff
@@ -84,42 +90,38 @@ impl StakeInfo {
                 .map(|vote_account| vote_account.activated_stake)
                 .sum();
 
-            if last_stake != (stake, total_stake) {
-                last_stake = (stake, total_stake);
-
-                // https://github.com/rpcpool/solana-private/blob/v1.18.18-triton/streamer/src/nonblocking/quic.rs#L780-L790
-                let max_streams = if stake == 0 || stake > total_stake {
-                    // https://github.com/rpcpool/solana-private/blob/v1.18.18-triton/streamer/src/nonblocking/stream_throttle.rs#L36-L76
-                    // `max_unstaked_connections` = 500, `max_streams_per_ms` = 250
-                    // `max_unstaked_load_in_throttling_window` = 0.2 * 250 * 100 / 500 = 10
-                    10
-                } else {
-                    // https://github.com/rpcpool/solana-private/blob/v1.18.18-triton/streamer/src/nonblocking/stream_throttle.rs#L151-L195
-                    // `max_unstaked_connections` = 500, `max_streams_per_ms` = 250
-                    // `current_load` = 2_500 (min value, give us mix streams capacity)
-                    // `capacity_in_ema_window` = 40_000 * stake (%)
-                    // `calculated_capacity` = 40_000 * stake (%) * 100 / 50 = 80_000 * stake (%)
-                    (80_000f64 * stake as f64 / total_stake as f64).floor() as u64
-                };
-
-                metrics::cluster_identity_stake_set(metrics::ClusterIdentityStakeKind::Jet, stake);
-                metrics::cluster_identity_stake_set(
-                    metrics::ClusterIdentityStakeKind::Total,
-                    total_stake,
-                );
-                metrics::cluster_identity_stake_set(
-                    metrics::ClusterIdentityStakeKind::MaxStreams,
-                    max_streams,
-                );
+            // https://github.com/rpcpool/solana-private/blob/v1.18.18-triton/streamer/src/nonblocking/quic.rs#L780-L790
```

Comment:
need to update these links to point to either rpcpool/solana-public or to anza-xyz/agave